packages/syft/src/syft/node/node.py

```diff
@@ -1435,26 +1435,36 @@ def create_default_worker_pool(node: Node) -> Optional[SyftError]:
 
     print("Building Default Worker Image")
 
-    # Build the Image for given tag
-    result = image_build_method(
-        context, image_uid=default_image.id, tag=DEFAULT_WORKER_IMAGE_TAG
-    )
-
-    if isinstance(result, SyftError):
-        print("Failed to build default worker image: ", result.message)
-        return
+    if not default_image.built_at:
```

Comment:
you can now call `if not default_image.is_built`

Reply:
Fixed in https://github.com/OpenMined/PySyft/pull/8380/commits/04d116caa158474f482206191f792ca499999520